go.mod

```diff
@@ -5,15 +5,25 @@ go 1.13
 require (
 	github.com/crossplaneio/crossplane v0.7.0-rc.0.20200206230838-4534223ff95e
 	github.com/crossplaneio/crossplane-runtime v0.4.1-0.20200201005410-a6bb086be888
-	github.com/google/go-cmp v0.3.1
-	github.com/pkg/errors v0.8.1
-	golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
+	github.com/google/go-cmp v0.4.0
+	github.com/pkg/errors v0.9.1
+	golang.org/x/net v0.0.0-20191028085509-fe3aa8a45271
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
 	gopkg.in/yaml.v2 v2.2.4
-	k8s.io/api v0.17.0
-	k8s.io/apimachinery v0.17.1
-	k8s.io/client-go v0.17.0
+	helm.sh/helm/v3 v3.0.0-20200205083830-5ec70ab27fbf
+	k8s.io/api v0.17.2
+	k8s.io/apiextensions-apiserver v0.17.2
+	k8s.io/apimachinery v0.17.2
+	k8s.io/client-go v0.17.2
 	sigs.k8s.io/controller-runtime v0.4.0
 	sigs.k8s.io/kustomize/api v0.3.0
 	sigs.k8s.io/yaml v1.1.0
 )
+
+replace (
+	// Helm requires these in this version. In master, they are not required,
```

Comment:
> In master, they are not required,

So in the `master` version of helm 3 they're not required anymore?

Reply:
Yes, it seems so. But I wanted to use a released version instead of a master commit. It's likely that we won't need these statements when we update the version of Helm3